send/generic_send.py

```diff
@@ -55,6 +55,17 @@
 		temp = tempfile.NamedTemporaryFile(mode="w+")
 		transport_command.extend(["-i", "-H", "Content-Type:application/x-tar", "-w", "%{http_code}", "--show-error",
 								  "--silent", "-o", temp.name, "-X", "PUT", "--data-binary", "@-"])
+		# Append optional BA credentials to URL destinations from standardized config
+		try:
```

Comment:
I'll probably rewrite this after #822 is reviewed and merged. In the end I'm not sure, if we need this in old `generic_send.py`, since I think we wanted to replace it with better implementation in `generic_sender.py`. So we might just say, who wants to use this feature needs to switch to using newer send script.